### PR TITLE
Remove redundant event_auth index

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

**Comment by NegativeMjark** [comment]
LGTM

**Comment by erikjohnston** [comment]
@NegativeMjark PTAL

File: synapse/storage/schema/delta/36/remove_auth_idx.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+DROP INDEX IF EXISTS current_state_events_state_key;
+DROP INDEX IF EXISTS current_state_events_type;
+
+DROP INDEX IF EXISTS transactions_have_ref;
+
+-- (topological_ordering, stream_ordering, room_id) seems like a strange index,
+-- and is used incredibly rarely.
+DROP INDEX IF EXISTS events_order_topo_stream_room;
+
+-- TODO: Drop event_auth stuff.
```
Comment (NegativeMjark) [inline synapse/storage/schema/delta/36/remove_auth_idx.py]: Is the TODO still valid?

Comment (erikjohnston) [inline synapse/storage/schema/delta/36/remove_auth_idx.py]: Oops, no. Done.

**Comment by erikjohnston** [comment]
@NegativeMjark PTAL

**Comment by NegativeMjark** [comment]
LGTM

**Pull request merged**